src/main/java/org/jenkinsci/plugins/fodupload/Utils.java

```diff
@@ -188,30 +190,9 @@ public static Boolean isUnauthorizedResponse(Response response) {
         return response.code() == HttpStatus.SC_FORBIDDEN || response.code() == HttpStatus.SC_UNAUTHORIZED;
     }
 
-    public static String createApplicationResponseViewModel(List<ApplicationApiResponse> applicationResponseList) {
+    public static <T> String createResponseViewModel(List<T> responseList) {
```

Comment:
Yeah, looks so much cleaner now, doesn't it? :)

Reply:
Mmhmm! Thanks you!